runtime/lua/vscode/internal.lua

```diff
@@ -340,6 +340,9 @@ function M.init_document_buffer(data)
   force_filetype()
   -- Set bufname before setting lines so that filetype detection can work ???
   api.nvim_buf_set_name(buf, data.bufname)
+  -- Let nvim resolve the physical path of our file to avoid relative path issues
+  -- with symbolic links when saving the buffer
```

Comment:
```suggestion
  -- with symbolic links when saving the buffer. #2284
```

Reply:
:+1: did not realize there was an issue open for this, good catch